molgenis-data/src/main/java/org/molgenis/data/decorator/DynamicRepositoryDecoratorRegistryImpl.java

```diff
@@ -125,9 +118,4 @@ public DynamicRepositoryDecoratorFactory getFactory(String id) {
       return emptyMap();
     }
   }
-
-  @EventListener
```

Comment:
Not entirely sure if it's an improvement to move the listening to the molgenis repository decorator factory. I kind of liked how the logic of whether the repository should be decorated or not was kept out of the Molgenis decorator factory and each decorator took care of its own logic.